scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  day TEXT,
+  start TEXT,
+  comments TEXT,
+  doctor_name TEXT,
+  patient_name text
+);
```

Comment:
the appointments table with the union should be created this way for references:

```sql
CREATE TABLE appointments(
  date TEXT, time TEXT, comment TEXT,
  doctor_id INTEGER REFERENCES doctors(id),
  patient_id INTEGER REFERENCES patients(id)
);
```